### PR TITLE
Added document for two TBv10 AVD diagnostic controls and Storage key rotation control.

### PR DESCRIPTION
**Pull request by HarshitArora8**

### CODE REVIEW

File: Control coverage/Feature/AVDApplicationGroups.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+### Rationale 
+Diagnostic logs are needed for creating activity trail while investigating an incident or a compromise.
+
+### Control Settings 
+```json 
+{
+    "DiagnosticForeverRetentionValue": "0",
+    "DiagnosticLogs": [
+        "Checkpoint",
+ 		"Management",
```
Comment (vaishnavipulluri) [inline Control coverage/Feature/AVDApplicationGroups.md:28]: indentation needs to be corrected

File: Control coverage/Feature/AVDApplicationGroups.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+	# Below commands will be useful to Configure diagnostic settings on AVDApplicationGroups
+
+    Connect-AzAccount
+	Set-AzContext -SubscriptionId "<sub id>"`
+	
+	#Add diagnostic settings to an existing AVDApplicationGroups
+	$log = @()
+	$log += New-AzDiagnosticSettingLogSettingsObject -Enabled $true -CategoryGroup allLogs
+
+	New-AzDiagnosticSetting -Name <DiagnosticSettingName> -ResourceId <AVDApplicationGroupsResourceId> -WorkspaceId <LogAnalyticsWorkspaceID> -Log $log -StorageAccountId <StorageAccountResourceId> -EventHubName <EventHubName>-EventHubAuthorizationRuleId <EventHubAuthorizationRuleId>
```
Comment (vaishnavipulluri) [inline Control coverage/Feature/AVDApplicationGroups.md:79]: Shall we also add option for enabling Retention Days?

File: Control coverage/Feature/AVDApplicationGroups.md (hunk 1 above)
Comment (HarshitArora8) [reply Control coverage/Feature/AVDApplicationGroups.md]: Fixed.

File: Control coverage/Feature/AVDApplicationGroups.md (hunk 2 above)
Comment (HarshitArora8) [reply Control coverage/Feature/AVDApplicationGroups.md:79]: Configuring retention days from diagnostic settings is not supported now. I have tried with Azure portal as well as PS commands but it clearly says not supported. please refer below screenshots as reference:

image

image

**Review by vaishnavipulluri — APPROVED** [review]

**Review by Aboli-msft — APPROVED** [review]

**Pull request merged**